grumblr/app/assets/javascripts/backbone/views/formView.js

```diff
@@ -0,0 +1,28 @@
+var FormView = Backbone.View.extend({
+  el: "#grumble-form",
+
+  events:{
+    "click .create": "markCreate"
+  },
+
+  initialize: function(){
+    console.log("form has been instatiated")
+    this.render();
+  },
+
+
+  render:function(){
+    var renderedHTML = HandlebarsTemplates['grumbles/form']();
+    this.$el.html(renderedHTML);
+  },
+
+  markCreate: function(){
```

Comment:
I'd prefer a clearer name for this function. Perhaps `createGrumble` instead of `markCreate`?